src/dualscreen.tsx

```diff
@@ -85,6 +86,18 @@ export class DualScreen extends BasePlugin<DualScreenConfig> implements IEngineD
     return new DualScreenEngineDecorator(engine, this, dispatcher);
   }
 
+  set _layout(layout: Layout) {
+    if(layout !== this.layout) {
+      this.layout = layout;
+      // @ts-expect-error - TS2339: Property 'dispatchEvent' does not exist on type 'KalturaPlayer'
```

Comment:
is this really required?

Reply:
yes it does, since this plugin relying on the old misconfigured TypeScirpt declarations